### DISCUSSION TITLE
Animated Scoreboard Or Scoreboard Title?

### DISCUSSION THREAD

**Discussion by ghost** [diamond-rip/Eden#397] [Ideas]
basically you would have certain variables or placeholders you can make into an animation and you place it throughout the scoreboard or the entire language.yml file instead of just copy pasting it would certainly also save time and straighten up the language.yml

**Comment by GoodestEnglish** [comment]
The reason why I made scoreboard so much line is because it allows user to fully customizable. Complex ideas like animation should support by other plugins, not Eden. 

Also, I would like to point out that new feature requests will mostly not going to be accepted until most of the bugs are fixed.

**Reply by ghost** [reply]
understandable but do you have any way to actually perform to make an animation using a different plugin it would be really useful to me I guess

**Reply by GoodestEnglish** [reply]
I don't think there is.
